src/controller/handler/router.go

```diff
@@ -120,6 +126,42 @@ func wrapper(inner http.HandlerFunc) http.HandlerFunc {
     })
 }
 
+func wrapperWS(inner func (string, int, []byte) ([]byte, error)) http.HandlerFunc {
```

Comment:
remove wrapperWS, move all to AgentConnect; add global var of `wsConns` func `WebsocketGetConnections` `WebsocketDisconnect` `WebsocketSendMessage`.